mkimage/debootstrap

```diff
@@ -203,10 +203,8 @@ fi
 	# delete all the apt list files since they're big and get stale quickly
 	rm -rf "$rootfsDir/var/lib/apt/lists"/*
 	# this forces "apt-get update" in dependent images, which is also good
-	if [ $suite != 'buster' ]; then
-		# Raspberry PI Org doesn't have buster yet.
-		echo "deb http://archive.raspberrypi.org/debian $suite main ui" >>  "$rootfsDir/etc/apt/sources.list.d/raspi.list"
-		chroot "$rootfsDir" bash -c 'sudo apt-key adv --batch --keyserver ha.pool.sks-keyservers.net  --recv-key 0x82B129927FA3303E'
-	fi
+	# Raspberry PI Org doesn't have buster yet.
```

Comment:
@nghiant2710 should remove this comment, as it's incorrect after these changes.

Reply:
updated!